python/src/aiconfig/eval/custom_eval/examples/travel/travel_eval.ipynb

```diff
@@ -104,63 +102,78 @@
     }
    ],
    "source": [
-    "print(\"\"\"\n",
+    "print(\n",
+    "    \"\"\"\n",
     "    Define test suite with inputs (option 1), \n",
     "      as opposed to using pre-computed AIConfig outputs (option 2)\n",
     "      \n",
     "      Define list of inputs and test criteria.\n",
     "      In this case, we are checking brevity for each test case\n",
     "      as well as checking that each output contains a specific expected substring.\n",
-    "\"\"\")\n",
-    "\n",
+    "\"\"\"\n",
+    ")\n",
     "\n",
     "\n",
-    "tsconfig = TestSuiteWithInputsSettings(\n",
-    "    {\n",
-    "        \"prompt_name\": \"get_activities\",\n",
-    "        \"aiconfig_path\": \"./travel_parametrized.aiconfig.json\",\n",
-    "    }\n",
+    "ts_settings = TestSuiteWithInputsSettings(\n",
+    "    prompt_name=\"gen_itinerary\",\n",
+    "    aiconfig_path=\"./travel_parametrized.aiconfig.json\",\n",
     ")\n",
     "\n",
     "\n",
-    "test_inputs = [\n",
-    "    \"Empire State Building is on fifth avenue. What is the cross street?\",\n",
-    "    \"What is the best borough?\",\n",
+    "test_inputs_with_substrings = [\n",
+    "    (\"Food\", \"Magnolia Bakery\"),\n",
+    "    (\"Iconic midtown skyscrapers\", \"Empire State Building\"),\n",
```

Comment:
Let's add comment here saying that these are in the form `<input, expected "output">`

Reply:
Well, that's sort of true. This is a special case of expected output where "expected" means the output contains this particular substring. I was hoping that would be self-documenting if you look down at the loop below - `substring_match(substring, case_sensitive=False)`.

Reply:
I'll explain the situation better.